user-manager-service/src/main/java/com/crmfoodestablishment/usermanager/crud/entity/User.java

```diff
@@ -0,0 +1,54 @@
+package com.crmfoodestablishment.usermanager.crud.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, unique = true)
+    private Long id;
+
+    @Column(name = "uuid", nullable = false, unique = true, updatable = false)
+    private UUID uuid;
+
+    @Column(name = "email", nullable = false, unique = true)
```

Comment:
length = 255? In intial-users-table size is specified. Also check other fields.

Reply:
resolved